### DISCUSSION TITLE
getFolderTrackCount plays file in that folder

### DISCUSSION THREAD

**Discussion by sferni** [Makuna/DFMiniMp3#150] [Q&A]
When i call the getFolderTrackCount method, a track from that folder is being played for a split second.
This also happens when i insert the SD Card.
I have a MH2024K-24SS from AZ Delivery, bought last week.
I noticed this also happens with other libraries, when i try to get the folder track count.

This is the debug output from the PlayRandom Sketch where i just commented out the dfmp3.playRandomTrackFromAll() and called getFolderTrackCount(0) exactly one time:

```
initializing...
OUT  7e ff 06 0c 01 00 00 fe ee ef
IN  7e ff 06 41 00 00 00 fe ba ef
IN  7e ff 06 3f 00 00 02 fe ba ef
SD Card, online
OUT  7e ff 06 46 00 00 00 fe b5 ef
OUT  7e ff 06 46 00 00 00 fe b5 ef
OUT  7e ff 06 46 00 00 00 fe b5 ef
version 0
OUT  7e ff 06 43 00 00 00 fe b8 ef
IN  7e ff 06 43 00 00 1e fe 9a ef
volume 30
OUT  7e ff 06 06 01 00 02 fe f2 ef
IN  7e ff 06 41 00 00 00 fe ba ef
OUT  7e ff 06 48 00 00 00 fe b3 ef
IN  7e ff 06 48 00 00 06 fe ad ef
files 6
OUT  7e ff 06 45 00 00 00 fe b6 ef
IN  7e ff 06 45 00 00 05 fe b1 ef
playback mode 5
starting...
OUT  7e ff 06 4e 00 00 00 fe ad ef
IN  7e ff 06 4e 00 00 02 fe ab ef
```

**Comment by Makuna** [comment]
The command to get playback mode is returning a value that isn't documented in any spec I have.  All the docs I have only have valid values up to 3 and yours is returning 5?
The documentation for this specific command (0x45) is sparse and often missing from chip to chip.  I suspect whatever chip version you have they expanded it.  This value may hint as some mode that autoplay?  Try setting the mode specifically using `setPlaybackMode()`, but even this command (0x08) can mean different things from chip to chip, some its set playback mode, some its repeat global track.

**Comment by Makuna** [comment]
From the output you provided, I see nothing to suggest that a request to play or even that a play finished happened.  It could just be an annoying issue with the chip.
